acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/PantheonNode.java

```diff
@@ -0,0 +1,167 @@
+
+    LOG.info("Pantheon Web3j service targeting: " + config.downstreamUrl());
+
+    this.jsonRpc =
+        new JsonRpc2_0Web3j(
+            new HttpService(config.downstreamUrl()),
+            config.pollingInterval().toMillis(),
+            Async.defaultExecutorService());
+
+    this.docker = docker;
+    pullPantheonImage();
+    this.pantheonContainerId = createPantheonContainer(config);
+    this.transaction = new Transactions(jsonRpc);
+    this.accounts = new Accounts(jsonRpc);
+  }
+
+  @Override
+  public void start() {
+    LOG.info("Starting Pantheon Docker container: " + pantheonContainerId);
+    docker.startContainerCmd(pantheonContainerId).exec();
+  }
+
+  @Override
+  public void shutdown() {
+    if (hasPantheonContainer()) {
+      stopPantheonContainer();
+      removePantheonContainer();
+    }
+  }
+
+  @Override
+  public void awaitStartupCompletion() {
+    LOG.info("Waiting for Pantheon to become responsive...");
+    waitFor(() -> assertThat(jsonRpc.ethBlockNumber().send().hasError()).isFalse());
+    LOG.info("Pantheon is now responsive");
+  }
+
+  @Override
+  public Accounts accounts() {
+    return accounts;
+  }
+
+  @Override
+  public Transactions transactions() {
+    return transaction;
+  }
+
+  private boolean hasPantheonContainer() {
+    return docker != null && pantheonContainerId != null;
+  }
+
+  private void stopPantheonContainer() {
+    try {
+      LOG.info("Stopping the Pantheon Docker container");
+      docker.stopContainerCmd(pantheonContainerId).exec();
+      docker.waitContainerCmd(pantheonContainerId).exec((new WaitContainerResultCallback()));
+    } catch (final NotModifiedException e) {
+      LOG.error("Pantheon Docker container has already stopped");
+    }
+  }
+
+  private void removePantheonContainer() {
+    LOG.info("Removing the Pantheon Docker container");
+    docker.removeContainerCmd(pantheonContainerId).withForce(true).exec();
+  }
+
+  private void pullPantheonImage() {
+    final String image = "pegasyseng/pantheon:latest";
+    final PullImageResultCallback callback = new PullImageResultCallback();
+    docker.pullImageCmd(image).exec(callback);
+
+    try {
+      LOG.info("Pulling the Pantheon Docker image...");
+      callback.awaitCompletion();
+      LOG.info("Pulled the Pantheon Docker image: " + image);
+    } catch (final InterruptedException e) {
+      LOG.error(e);
+    }
+  }
+
+  private String createPantheonContainer(final NodeConfiguration config) {
+    final HostConfig portBindingConfig =
+        HostConfig.newHostConfig().withPortBindings(tcpPortBinding(config), wsPortBinding(config));
+
+    try {
+      final CreateContainerCmd createPantheon =
+          docker
+              .createContainerCmd("pegasyseng/pantheon:latest")
+              .withHostConfig(portBindingConfig)
+              .withCmd(
+                  "--miner-enabled",
+                  "--miner-coinbase",
+                  "fe3b557e8fb62b89f4916b721be55ceb828dbd73",
```

Comment:
could use the public key from Accounts.GENESIS_ACCOUNT_ONE_PUBLIC_KEY here

Reply:
🤷‍♂️ any reason? ...the genesis account really does not need the money. As a seperate account, in theory mining rewards could be tested in isolation (as the genesis account would have to pay for sending transactions)